bundles/org.openhab.binding.linuxinput/src/main/java/org/openhab/binding/linuxinput/internal/LinuxInputHandler.java

```diff
@@ -95,9 +95,13 @@ boolean delayedSetup() throws IOException {
         EvdevDevice newDevice = new EvdevDevice(config.path);
         for (EvdevDevice.Key o : newDevice.enumerateKeys()) {
             String name = o.getName();
+            if (name == null) {
+                name = Integer.toString(o.getCode());
```

Comment:
Are you certain that `o.getCode()` always represent an integer ?
Catching the exception could be more secure.

Reply:
I'm not sure I understand.
Integers are how all events are represented by the kernel and libevdev.

`getCode()` is guaranteed to return an integer:
https://github.com/openhab/openhab-addons/blob/main/bundles/org.openhab.binding.linuxinput/src/main/java/org/openhab/binding/linuxinput/internal/evdev4j/EvdevDevice.java#L211